py/selenium/webdriver/remote/client_config.py

```diff
@@ -205,7 +209,10 @@ def auth_type(self) -> str:
     @auth_type.setter
     def auth_type(self, value: str) -> None:
         """Sets the type of authentication to the remote server if it is not
-        using basic with username and password."""
+        using basic with username and password.
+
+        Support values: Bearer, X-API-Key. For others, please use `extra_headers` instead
```

Comment:
Should we be having these as constants that people can pass in? It will prevent typos and give people examples of what to us